frameworks/PHP/clancats/setup.sh

```diff
@@ -0,0 +1,11 @@
+export PHP_HOME=${IROOT}/php-5.5.17
+export COMPOSER_HOME=${IROOT}/php-composer
+export PHP_FPM=${PHP_HOME}/sbin/php-fpm
+export NGINX_HOME=${IROOT}/nginx
+
+sed -i 's|localhost|'"${DBHOST}"'|g' index.php
+sed -i 's|root .*/FrameworkBenchmarks|root '"${TROOT}"'|g' deploy/nginx.conf
```

Comment:
For some reason the /; was causing it never to match but other than that I made the change.